api-audit/src/main/java/com/capitalone/dashboard/response/AuditReviewResponse.java

```diff
@@ -1,10 +1,10 @@
 package com.capitalone.dashboard.response;
 
-import java.util.HashSet;
-import java.util.Set;
+import java.util.SortedSet;
```

Comment:
refrain from changing signature, sorting logic needs to be implemented in Fongo Unit Tests